packages/webcomponents/src/components/form/form-control-file.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+  Component,
+  Host,
+  h,
+  Prop,
+  Event,
+  EventEmitter,
+  State,
+  Element,
+} from '@stencil/core';
+
+@Component({
+  tag: 'form-control-file',
+  styleUrl: 'form-control-file.scss',
+  shadow: true,
+})
+export class FileInput {
+  @Element() el: HTMLElement;
+  @Prop() label: string;
+  @Prop() name: any;
```

Comment:
we probably want  `error`, `disabled` and maybe `label` to be optional

Reply:
@TiagoFuelber 

Hmmm, I'm not sure that we need to do that since its written the same way in each of the other form-control components. 

I wonder if that's something we should fix in each of the form-control components, though I'm not entirely sure why we have never gotten warnings about this before. 🤔